docs/core/upgrade.md

```diff
@@ -64,20 +50,38 @@ Migrations are run inside of an `UpgradeHandler` using  `app.mm.RunMigrations(ct
 
 ```go
 cfg := module.NewConfigurator(...)
-app.UpgradeKeeper.SetUpgradeHandler("my-plan", func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+app.UpgradeKeeper.SetUpgradeHandler("my-plan", func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
 
     // ...
     // do upgrade logic
     // ...
 
-    // RunMigrations returns the VersionMap
-    // with the updated module ConsensusVersions
-    return app.mm.RunMigrations(ctx, vm)
+    orderedVersions = module.DefaultMigrationsOrder(fromVM)
```

Comment:
I will update the rest of the documentation later, once we agree on the solution. 

Here is the essential part. In `SetUpgradeHandler` we need to order all migrations. Below you will find implementation for the default function.